### ISSUE TITLE
Copy paste from Web  LLM mode 

### ISSUE THREAD

**Issue by AriMKatz** [zed-industries/zed#21820]
### Check for existing issues

- [X] Completed

### Describe the feature

https://x.com/paulgauthier/status/1866519827814428846?t=nZ87I9E19ecffd_6k7i44g&s=19

Basically, copy your conversation with o1, Claude online etc and have a cheap local LLM apply ot

### Environment

Windows

**Comment by notpeter** [comment]
I don't think this is the direction we're looking to go at the moment. Thanks for filing.

**Comment by AriMKatz** [comment]
Understood! I'll just add that o5 pro is substantially better than other models and is not available by API, so I feel like that's a compelling reason to have easy context export and import
